contractcourt/channel_arbitrator.go

```diff
@@ -787,7 +787,7 @@ func (c *ChannelArbitrator) relaunchResolvers(commitSet *CommitSet,
 		// TODO(roasbeef): this isn't re-launched?
 	}
 
-	c.launchResolvers(unresolvedContracts)
+	c.launchResolvers(unresolvedContracts, true)
```

Comment:
IIUC this is the only time the `immediate` is *ever* `true` in the context of the contract resolvers.

Reply:
Yea, this is the restart scenario.